### ISSUE TITLE
NumberFormatException when message is sent from 

### ISSUE THREAD

**Issue by viz123** [mrniko/netty-socketio#223]
I am trying the ChatLauncher demo of netty-socketio. But, I am using SocketIOClientDotNet as the client. Please see https://github.com/Quobject/SocketIoClientDotNet

I am able to connect to the server. But when I send a json object from SocketIOClientDotNet , I get a NumberFormatException on the server side (ChatLauncher). It looks something like this:

....java.lang.NumberFormatException: Invalid int: "���42["chatevent",{"userName"" at java.lang.Integer.invalidInt(Integer.java:138).....

The same code from SocketIOClientDotNet is working fine with socket-io running on a node.js server.

While I can assume that the client is sending data in wrong format, but the fact that it works with node.js makes me think that the issue may be handled on the server side too.

I can send the test code if needed.

**Comment by mrniko** [comment]
> is working fine with socket-io running on a node.js server

which version is it?

**Comment by viz123** [comment]
Here are the version details:

SocketIoClientDotNet: Latest version (I did not find any version number there)
Node.js : 0.12.0 running on Windows 8 64 bit.
socket.io: 1.3.5
netty-socketio: latest (1.7.7 )

**Comment by mrniko** [comment]
I don't understand the .NET :) Could you provide me a test with easy way to run using SocketIoClientDotNet?

**Comment by viz123** [comment]
I uploaded a zip file at the following location: https://www.dropbox.com/s/pv4mcxqy8dcsnqj/Release.zip?dl=0

It requires .net 4.5. 

Among other things, there are files named test_data.txt and dotnettest.exe

put your URL and test object in test_data.txt and then execute dotnettest.exe from command prompt.

I noticed that the error is coming intermittently.

Let me know if I can help in any other way. Thanks.
